doc/MANUAL.md

```diff
@@ -208,6 +210,10 @@ See the detailed description of [`--input-file`](#-i-input_file---input-file-inp
 
 This option controls the frequency of updates for the CVE data from the National Vulnerability Database.  By default, the tool checks the staleness of the data with every run, and if the data is more than one day old, it gets an update from NVD.  You may also choose to update the data `now` (in which case all cached data is deleted and a full new download is done) or `never` in which case the staleness check is not done and no update is requested.  The `now` and `never` modes can be combined to produce alternative update schedules if daily is not the desired one.
 
+### -n, --nvd 
+
+This option allows you to use the updated NVD CVE Retrieval API. You can read more about this [here](https://csrc.nist.gov/CSRC/media/Projects/National-Vulnerability-Database/documents/web%20service%20documentation/Automation%20Support%20for%20CVE%20Retrieval.pdf)
+
```

Comment:
I think we need a little bit more documentation here.  
1. Why would one want to use this API?  
2. What are the performance impacts (some example numbers/percentages would be good)
3. Does this use less bandwidth?
4. Are there any other implications users should know about?

Assume the user isn't going to read the paper (although it's definitely good to link) and is just going to decide "should I use this flag?" based on what's in the doc.